File: Chapter 04/C Code/4_10.c
#include<stdio.h>
void main()
{
int i, v=0;
char s[30];
printf("Enter a string: ");
gets(s);
i=0;
while(s[i] != '\0'){
if ((((int)(s[i]) >=65) && ((int)(s[i]) <=90)) || (((int)(s[i]) >=97) && ((int)(s[i]) <=122)))
{

if (s[i] != 'a' && s[i] != 'e' && s[i] != 'i' && s[i] != 'o' && s[i] != 'u')
    {v=v+1;}
i=i+1;
}
else{
    printf("Special character encountered.\n");
    break;
}
}
printf("There are %d consonants in the string.", v);
}

File: Chapter 04/Python code/4_10.py
s= input("Enter a string: ")
v=0
for i in s:
    if ord(i) in range(65,91) or ord(i) in range(97,123):
        if(i != 'a' and i != 'e' and i != 'i' and i != 'o' and i != 'u'):
            v=v+1
    else:
        print("Special character encountered. ")
        break
print ("There are ", v, "consonants in the string.")
